diff -up kcoreaddons-master/src/lib/text/kfuzzymatcher.h.omv~ kcoreaddons-master/src/lib/text/kfuzzymatcher.h
--- kcoreaddons-master/src/lib/text/kfuzzymatcher.h.omv~	2023-08-02 17:08:35.854164303 +0100
+++ kcoreaddons-master/src/lib/text/kfuzzymatcher.h	2023-08-02 17:09:15.964294525 +0100
@@ -12,6 +12,7 @@
 #include <kcoreaddons_export.h>
 
 #include <QtContainerFwd>
+#include <QtCore/qtypeinfo.h> // for Q_PRIMITIVE_TYPE
 
 class QString;
 class QStringView;
